tetris.py

```diff
@@ -165,8 +146,31 @@ def run_gui():
 
     root.title("Tetris – by The Philgrim, Arrinao, and Master Akuli")
     # root.iconphoto(False, tkinter.PhotoImage(file=image_name.png")) TODO: INSERT LATER
+
     root.mainloop()
 
+# Draws the board consisting of 15x10 squares before the game starts
```

Comment:
This should be a docstring, not a comment. But there's already a docstring below.

```suggestion
```